combat-trainer.lic

```diff
@@ -1262,6 +1262,7 @@ class SpellProcess
     end
 
     snapshot = DRSkill.getxp(@skill)
+    check_face(game_state) if DRRoom.dead_npcs.size > 0 && DRStats.empath?
```

Comment:
💡 You have the `DRRoom.dead_npcs.size > 0` check here and inside the function. I suggest consolidating all of the conditionals inside the function.

Reply:
That one is inside the spells section. Wasnt able to figure out how to call check face from combat class inside spell class.
I will change it so the check is inside the function instead though.